lib/desired-caps.js

```diff
@@ -0,0 +1,102 @@
+import log from './logger';
+import validator from 'validate.js';
+
+validator.validators.isString = function(value) {
```

Comment:
we do `function (value) {`

i'm also surprised that these basic type validations aren't built into the library